requirements.txt

```diff
@@ -1,10 +1,13 @@
+# No setup.py, unable to install using pip
+git+https://github.com/pymedusa/adba.git@119b9d30a30feb97bcea5e1dc742c82340300091#egg=adba
```

Comment:
FYI, since the `adba` package has no `setup.py`, it can't be installed using pip, that's why I decided to comment it out and added a comment above.

Reply:
Ooh yeah didnt think of that